build/repeatmasker.sh

```diff
@@ -0,0 +1,36 @@
+sudo apt-get -y install perl-modules cpanminus python3-h5py
```

Comment:
Can you move this to `cluster/images/build/repeatmasker.sh` please? Top-level directories should be added judiciously and I feel as though having it nearer the rest of the scripts will help make it clear what it's for.

Also maybe just some comments at the top to indicate what it's for OR an amendment to the `cluster/images/README.md` file so people know they need to run it when the OS/major dependencies change but not on a regular basis, would be useful.

Reply:
Added to cluster/images/build subdirectory. This still keeps it seperate from the image building scripts, but doesn't introduce a new top-level directory